src/content/docs/en/guides/prefetch.mdx

```diff
@@ -114,7 +114,14 @@ prefetch('/about', { with: 'fetch' });
 prefetch('/about', { with: 'link' });
 ```
 
-The `prefetch()` API includes the same [data saver mode](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/saveData) and [slow connection](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType) detection, so that it only prefetches when needed.
+The `prefetch()` API includes the same [data saver mode](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/saveData) and [slow connection](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType) detection, with fallback to the `tap` method.
```

Comment:
Hi @sarah11918, to clarify, the `prefetch()` API is independent of tap/hover/viewport strategies as it is manually called by the user in a function. So, manually calling `prefetch()` can't fall back to tap, as it is not caused by user interaction like hover or viewport. I suggest we keep it as is for now, indicating that it bails on slow connections, and you can opt out of this behavior with the `ignoreSlowConnection` flag.

If "only prefetching when needed" seems vague, we can rephrase it to specify that it will bail on slow connections. Let me know what you think!

Reply:
Gotcha! In that case, then nope, I'm cool with the original! We can revert. Thanks for clarifying. Just wanted to make sure this wasn't overlooked.

Reply:
Yup agree with Smit here!